redfish-core/lib/service_root.hpp

```diff
@@ -123,6 +123,34 @@ inline void
     asyncResp->res.jsonValue["Oem"]["IBM"]["DateTimeLocalOffset"] =
         redfishDateTimeOffset.second;
 
+    // Report if ACFWindowActive
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    const std::variant<bool>& retVal) {
+            if (ec)
+            {
+                BMCWEB_LOG_ERROR
+                    << "Failed to read ACFWindowActive property";
```

Comment:
Your call if you want an internal error, it more common to have one, but I guess someone could argue this isn't an important property? What would the GUI do if this isn't there?

Reply:
Intended usage is: if the ACFWindowActive property is present and has the true value, then show the "Upload ACF" button on the login page (before authentication and before a session is created).

Reply:
I'll change this to default value false.